lib/chef/knife/bootstrap.rb

```diff
@@ -18,11 +18,19 @@
 
 require "chef/knife"
 require "chef/knife/data_bag_secret_options"
+require "erubis"
+require "chef/knife/bootstrap/chef_vault_handler"
+require "chef/knife/bootstrap/client_builder"
+require "chef/util/path_helper"
+require "chef/dist"
+require "license_acceptance/cli_flags/mixlib_cli"
+# require "license_acceptance/acceptor"
```

Comment:
Can we remove this?

Reply:
This should be un-commented because we use the acceptor below